stock_move_packaging_qty/__manifest__.py

```diff
@@ -8,9 +8,11 @@
     "website": "https://github.com/OCA/stock-logistics-warehouse",
     "category": "Warehouse",
     "depends": ["stock"],
+    "development_status": "Alpha",
```

Comment:
Be careful as if there are some depending modules on this, they are becoming Alpha too (and possible errors in pre-commit).